include/readdy/api/SimulationLoop.h

```diff
@@ -250,13 +252,13 @@ class SimulationLoop {
     void run(const continue_fun &continueFun) {
         validate(_timeStep);
         {
-            bool requiresNeighborList = _forces || _reactions || _topologyReactions;
+            bool requiresNeighborList = kernel()->context().calculateMaxCutoff() > 0;
```

Comment:
I think a neighbor list is always required when the neighbor list cutoff is > 0 ... which should probably default to whatever calculateMaxCutoff gives (perhaps can be implemented as an `std::optional`?) but also be overridden.

Reply:
this flag is now determined from the `initNeighborList->cutoffDistance()`, which is at least the context's maxcutoff, but might have been changed between construction of the loop and the run.